code/modules/power/supermatter/supermatter.dm

```diff
@@ -519,9 +519,11 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 /obj/machinery/power/supermatter_crystal/attack_tk(mob/user)
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
-		to_chat(C, "<span class='userdanger'>That was a really dumb idea.</span>")
-		var/obj/item/bodypart/head/rip_u = C.get_bodypart(BODY_ZONE_HEAD)
-		rip_u.dismember(BURN) //nice try jedi
+		to_chat(C, "<span class='userdanger'>That was a rea-</span>")
```

Comment:
I kind of feel like this isn't as comedic as the old one, like you wouldn't really get it unless you had seen the old one

Reply:
I can cut off the very end if you think that'd work better, or just leave the original.

Reply:
either would probably be better

Reply:
fixed.